File: android/app/src/main/kotlin/com/goalmaker/app/application/planning/AreaItem.kt
package com.goalmaker.app.application.planning

/**
 * An area as lists and chips show it; [colorId] names a color in the area palette (themes.json). An
 * [archived] area keeps its tasks and chips but leaves the pickers and filters (docs/lists.md).
 */
data class AreaItem(
    val id: String,
    val name: String,
    val colorId: String,
    val emoji: String?,
    val archived: Boolean = false,
)

File: windows/src/GoalMaker.Core/Planning/AreaItem.cs
namespace GoalMaker.Core.Planning;

/// <summary>
/// An area as lists and chips show it; <paramref name="ColorId"/> names a color in the area palette
/// (themes.json). An <paramref name="Archived"/> area keeps its tasks and chips but leaves the pickers
/// and filters (docs/lists.md).
/// </summary>
public sealed record AreaItem(string Id, string Name, string ColorId, string? Emoji, bool Archived = false);
